pkg/cedar/intake/models/easi_intake.go

```diff
@@ -1,53 +1,66 @@
+	CostIncrease                string               `json:"costIncrease" jsonschema:"description=Is there a cost increase associated with this request,example=YES,example=NOT_SURE"`
+	CostIncreaseAmount          *string              `json:"costIncreaseAmount,omitempty" jsonschema:"description=How much is the cost increase,example=Over two million dollars"`
+	DecidedAt                   *string              `json:"decidedAt,omitempty" jsonschema:"description=Timestamp of when decision was reached,example=2022-02-17T14:34:43Z"`
+	DecisionNextSteps           *string              `json:"decisionNextSteps,omitempty" jsonschema:"description=Steps that the business owner should take after receiving their decision,example=Go get a contract"`
+	EaCollaborator              *string              `json:"eaCollaborator,omitempty" jsonschema:"description=N/A,example=N/A"` // TODO: doesn't seem like this is ever populated, remove?
+	EaCollaboratorName          *string              `json:"eaCollaboratorName,omitempty" jsonschema:"description=Enterprise Architecture (EA) Collaborator,example=John Doe"`
+	EaSupportRequest            *bool                `json:"eaSupportRequest,omitempty" jsonschema:"description=Does the request need EA support,example=True"`
+	ExistingContract            string               `json:"existingContract" jsonschema:"description=Is there an existing contract for this effort,example=HAVE_CONTRACT"`
+	ExistingFunding             *bool                `json:"existingFunding,omitempty" jsonschema:"description=Will this project be funded out of an existing source,example=True"`
+	FundingNumber               *string              `json:"fundingNumber,omitempty" jsonschema:"description=six digit funding number,example=123456"`
+	FundingSource               *string              `json:"fundingSource,omitempty" jsonschema:"description=Source of funding,example=Prog Ops"`
+	FundingSources              []*EASIFundingSource `json:"fundingSources,omitempty" jsonschema:"description=Array of funding sources, which contain a source of funding and a six-digit funding number,example=N/A"`
+	GrbDate                     *string              `json:"grbDate,omitempty" jsonschema:"description=Scheduled date for the Governance Review Board (GRB) meeting,example=2025-12-12T00:00:00Z"`
+	GrtDate                     *string              `json:"grtDate,omitempty" jsonschema:"description=Scheduled date for the Governance Review Team (GRT) meeting,example=2025-10-20T00:00:00Z"`
+	GrtReviewEmailBody          *string              `json:"grtReviewEmailBody,omitempty" jsonschema:"description=N/A,example=N/A"` // TODO: not sure when/if this is populated
+	Isso                        *string              `json:"isso,omitempty" jsonschema:"description=N/A,example=N/A"`               // TODO: doesn't seem like this is ever populated, remove?
```

Comment:
Looks like these are being re-introduced from an old PR that removed them https://github.com/CMSgov/easi-app/pull/1668